new-integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/impl/Domain.java

```diff
@@ -89,7 +94,36 @@ public static boolean adminT3ChannelAccessible(String domainUid, String namespac
     return true;
   }
 
-  public static boolean adminNodePortAccessible(String domainUid, String namespace) {
+  /**
+   * Login to the WebLogic console and validate its the Home page.
```

Comment:
modify the description to indicate that this assertion is to check node port is working ...

Verify admin node port(default/t3channel) is accessible by login to WebLogic console using the node port and validate its the Home page.

Reply:
Thanks for the write up. I modified it.